dist/raml-parser.js

```diff
@@ -10950,6 +10950,8 @@ else {
     ['this',     'these'],
     ['that',     'those'],
     // Words ending in with a consonant and `o`.
+    ['echo', 'echoes'],
```

Comment:
Where do these come from? is it an updated dependency? we should put this in a separate commit probably.

Also that dependency is not embedding the version numbers in a header.